pkg/apis/batch/validation/validation.go

```diff
@@ -490,11 +494,14 @@ func validateConcurrencyPolicy(concurrencyPolicy *batch.ConcurrencyPolicy, fldPa
 	return allErrs
 }
 
-func validateScheduleFormat(schedule string, timeZone *string, fldPath *field.Path) field.ErrorList {
+func validateScheduleFormat(schedule, oldSchedule string, timeZone *string, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if _, err := cron.ParseStandard(schedule); err != nil {
 		allErrs = append(allErrs, field.Invalid(fldPath, schedule, err.Error()))
 	}
+	if !strings.Contains(oldSchedule, "TZ") && strings.Contains(schedule, "TZ") {
+		allErrs = append(allErrs, field.Invalid(fldPath, schedule, "cannot use TZ or CRON_TZ in schedule, use timeZone instead"))
+	}
 	if strings.Contains(schedule, "TZ") && timeZone != nil {
 		allErrs = append(allErrs, field.Invalid(fldPath, schedule, "cannot use both timeZone field and TZ or CRON_TZ in schedule"))
```

Comment:
Should we update this error message like `cannot use both timeZone field and TZ or CRON_TZ in schedule, and use timeZone instead.`

Reply:
yes, tell them to use the timeZone field

also, update `cronJobStrategy#WarningsOnUpdate` to push to use the timeZone field